### ISSUE TITLE
GMT_seamount_cum_inc.sh crashes

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/gmt#3726]
```
$ sh GMT_seamount_cum_inc.sh
ERROR: Caught signal number 11 (Segmentation fault) at
0   supplements.so                      0x0000000113bd5f0d GMT_grdseamount + 5101
1   ???                                 0x0000000000000000 0x0 + 0
Stack backtrace:
0   libgmt.6.dylib                      0x000000010dd9e905 sig_handler + 581
1   libsystem_platform.dylib            0x00007fff710185fd _sigtramp + 29
2   ???                                 0x0000000000000000 0x0 + 0
3   libgmt.6.dylib                      0x000000010ddc0800 GMT_Call_Module + 2160
4   gmt                                 0x000000010dd8fa9e main + 2462
5   libdyld.dylib                       0x00007fff70e1fcc9 start + 1
ERROR: Caught signal number 11 (Segmentation fault) at
0   supplements.so                      0x000000010beabf0d GMT_grdseamount + 5101
1   ???                                 0x0000000000000000 0x0 + 0
Stack backtrace:
0   libgmt.6.dylib                      0x0000000107b2c905 sig_handler + 581
1   libsystem_platform.dylib            0x00007fff710185fd _sigtramp + 29
2   ???                                 0x0000000000000000 0x0 + 0
3   libgmt.6.dylib                      0x0000000107b4e800 GMT_Call_Module + 2160
4   gmt                                 0x0000000107b21a9e main + 2462
5   libdyld.dylib                       0x00007fff70e1fcc9 start + 1
GMT_seamount_cum_inc.sh: line 15: c.lis: No such file or directory
GMT_seamount_cum_inc.sh: line 21: i.lis: No such file or directory
subplot [ERROR]: No subplot information file!
end [WARNING]: subplot was never completed - plot items in last panel may be missing
```

**Comment by PaulWessel** [comment]
Thanks, it runs manually but crases with out any feedback for me on macos. Will look after swim.

**Comment by PaulWessel** [comment]
SO I check out master, build and run your command with sh and it runs fine.  When I run ctest -R GMT_seamount_cum_inc.sh then it fails and the log says

```
Start testing: Jul 25 10:40 HST
----------------------------------------------------------
116/908 Testing: GMT_seamount_cum_inc.sh
116/908 Test: GMT_seamount_cum_inc.sh
Command: "/opt/local/bin/bash" "gmtest" "GMT_seamount_cum_inc.sh"
Directory: /Users/pwessel/GMTdev/gmt-dev/dbuild/doc/scripts
"GMT_seamount_cum_inc.sh" start time: Jul 25 10:40 HST
Output:
----------------------------------------------------------
Set GMT_SESSION_NAME = 67183
ERROR: ./GMT_seamount_cum_inc.sh:14
memtrack errors: 0
exit status: 1
<end of output>
Test time =   1.66 sec
----------------------------------------------------------
Test Failed.
"GMT_seamount_cum_inc.sh" end time: Jul 25 10:40 HST
"GMT_seamount_cum_inc.sh" time elapsed: 00:00:01
----------------------------------------------------------

End testing: Jul 25 10:40 HST
```

what is different here?  If I could get a crash I can debug.

**Comment by seisman** [comment]
Sorry, I think I forgot to build the latest master branch when I ran the script.

The nightly CI jobs report the same message as what you just showed.

**Comment by PaulWessel** [comment]
I suspect it is the shell for loop that in ctest somehow gets split up.  I will run the 4 commands manually and see if that fixes it.

**Comment by PaulWessel** [comment]
Should be fixed in #3727 - please close once that is confirmed.

**Comment by seisman** [comment]
Confirmed. Closing.
